internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -54,14 +54,21 @@ type prDetails struct {
 	PrNumber  *int
 }
 
-// NewCircleCiService creates a new CircleCi service
-func NewCircleCiService(token string) diffreviewer.DiffReviewer {
+// NewAuthenticatedCircleCiService creates a new CircleCi service with an authenticated Github client
+func NewAuthenticatedCircleCiService(token string) diffreviewer.DiffReviewer {
```

Comment:
nit - can we update this naming to make it a little clearer that the only real difference is whether we write comments to github PR or not? Maybe like `NewCircleCIService` and `NewCircleCIServiceWithGHComments` or something?